addons/pos_self_order/static/src/app/components/product_card/product_card.xml

```diff
@@ -6,7 +6,7 @@
             t-att-title="props.product.display_name"
             t-on-click="() => this.selectProduct()"
             t-ref="selfProductCard">
-            <div t-if="props.product.description_self_order" class="product-information-tag" t-on-click.prevent.stop="showProductInfo">
+            <div t-if="props.product.description_self_order and props.product.description_self_order != '&lt;p>&lt;br>&lt;/p>'" class="product-information-tag" t-on-click.prevent.stop="showProductInfo">
```

Comment:
```suggestion
            <div t-if="!this.isHtmlEmpty" class="product-information-tag" t-on-click.prevent.stop="showProductInfo">
```

use this method
```
    get isHtmlEmpty() {
        // Security: is ok to use this there since the description_self_order is already trusted.
        const div = Object.assign(document.createElement("div"), {
            innerHTML: this.props.product.description_self_order,
        });
        return div.innerText.trim() === "";
    }
```